python/lsst/ap/verify/pipeline_driver.py

```diff
@@ -183,6 +182,29 @@ def runApPipeGen3(workspace, parsedCmdLine, processes=1):
         log.info('Skipping AP pipeline entirely.')
 
 
+def _getApdbArguments(workspace):
+    """Return the config options for running make_apdb.py on this workspace,
+    as command-line arguments.
+
+    Parameters
+    ----------
+    workspace : `lsst.ap.verify.workspace.WorkspaceGen2`
+        A Workspace whose config directory may contain an
+        `~lsst.ap.pipe.ApPipeTask` config.
+
+    Returns
+    -------
+    args : `list` of `str`
+        Command-line arguments calling ``--config`` or ``--configFile``,
```

Comment:
Errant `configFile` camel case when you meant `configfile`? Please no third standard 😭